cmd/dm-ctl/main.go

```diff
@@ -23,17 +23,24 @@ import (
 	"syscall"
 
 	"github.com/chzyer/readline"
-	"github.com/pingcap/dm/pkg/log"
-	"github.com/pingcap/errors"
-
 	"github.com/pingcap/dm/dm/ctl"
 	"github.com/pingcap/dm/dm/ctl/common"
+	"github.com/pingcap/dm/pkg/log"
 	"github.com/pingcap/dm/pkg/utils"
+	"github.com/pingcap/errors"
```

Comment:
putting this line in the above section is better, seems it does not belong to _dm_ but only to _pingcap_.

Reply:
Done [here](https://github.com/pingcap/dm/pull/287/files#diff-61075abbd9aac2de49a1fe702632b302L27)